component/argocd.jsonnet

```
@@ -7,12 +8,14 @@ local params = inv.parameters.argocd;
 local common = import 'common.libsonnet';
 local isOpenshift = std.startsWith(params.distribution, 'openshift');
 
+local resync_string = if std.get(params, 'resync_seconds', 0) > 0 then std.format('%dm0s', std.mod(params.resync, 60)) else params.resync_time;
```

Might make sense to also print a deprecation warning:

```suggestion
local resync_string =
  if std.get(params, 'resync_seconds', 0) > 0 then
    std.trace(
      'Parameter `resync_seconds` is deprecated. Please update your config to use `resync_time`',
      std.format('%dm0s', std.mod(params.resync, 60)),
    )
  else
    params.resync_time;
```